src/ResourceManager/Storage/Commands.Management.Storage/Commands.Management.Storage.csproj

```diff
@@ -135,12 +136,6 @@
     <Reference Include="System.Management.Automation, Version=3.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL" />
   </ItemGroup>
   <ItemGroup>
-    <Compile Include="..\..\..\Common\Commands.Common.Storage\Adapters\ARM.Storage.3\ARMStorageProvider.cs">
```

Comment:
As there are no one else need to use SRP SDK 5.0, and we are on a tight schedule for the PR, implement the adapter interface for the new storage version might NOT be our priority now.
We can write an adapt in the future when someone else need that.